src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskCompleteByIdControllerFailureTest.java

```diff
@@ -506,59 +541,56 @@ void beforeEach() {
                 .thenReturn(false);
         }
 
-        @Test
-        void should_return_500_with_application_problem_response_when_task_update_call_fails() throws Exception {
```

Comment:
Another one? This covers when we fail to complete and fails to update the task. Are covered on cftTaskDb side?

Reply:
done